xformers/csrc/attention/hip_fmha/generate_instances.py

```diff
@@ -6,11 +6,13 @@
 #
```

Comment:
Add a manual on how to use this generate_instances.py?

Reply:
Just added a `GENERATE_INSTANCES.md`